src/index.ts

```diff
@@ -53,19 +62,59 @@ export default class DataSource {
    *
    * @param opts
    */
-  async getMultipleTargets(opts: any = {}) {
+  async getMultipleTargets(sets: any = [], opts: any = {}) {
```

Comment:
Doesn't this param addition mean that the current way SD's are using this method has to somewhat change.
As the release we did in `0.0.3` we wanted to make it consistent to how we did `CD.getCORS`. 

Which was something like this: 

```js
const options = {
  method: 'POST',
  body: JSON.stringify([{}])
}
getMultipleTargets(options)
```

Would it make more sense to make `sets` the second argument and making it optional?

Reply:
Yeah that makes sense. I'll revert it to how it was before (just taking in a single arg `options`).
I'm not sure that having `sets` as the second optional arg adds any value, do we even need it then?

Reply:
Should I also update `getSingleTarget` to also just take a single arg `options`?

Reply:
Yea, I wasn't sure if you needed it for something else. 
I do agree keeping it as a single arg `options` to make it similar to how we doing `getMultipleTargets` would be better for consistency